frontend/src/js/query-runner/actions.ts

```diff
@@ -179,12 +178,7 @@ const getQueryErrorMessage = ({
     return t("queryRunner.queryCanceled");
   }
 
-  return (
-    (error &&
-      error.code &&
-      getExternalSupportedErrorMessage(t, error.code, error.context)) ||
-    t("queryRunner.queryFailed")
-  );
+  return error?.message || t("queryRunner.queryFailed");
```

Comment:
That's the main change in the logic: now trying to use `message` if it's there, otherwise use a generic query failed message